src/storage-preview/azext_storage_preview/__init__.py

```diff
@@ -128,7 +127,7 @@ def register_common_storage_account_options(self):
             encryption_choices = list(
                 t_encryption_services._attribute_map.keys())  # pylint: disable=protected-access
             self.argument('encryption_services', arg_type=get_enum_type(encryption_choices),
-                          resource_type=CUSTOM_MGMT_STORAGE, min_api='2016-12-01', nargs='+',
+                          resource_type=CUSTOM_MGMT_PREVIEW_STORAGE, min_api='2016-12-01', nargs='+',
```

Comment:
Not sure if it's necessary to change `min_api`

Reply:
same as above